src/stretch/agent/robot_agent.py

```diff
@@ -1361,7 +1361,15 @@ def move_to_nav_posture(self):
         """Move the robot to navigation posture."""
         self.robot.move_to_nav_posture()
 
-    def print_found_classes(self, goal: Optional[str] = None):
+    def is_match_by_feature(self, instance, goal):
```

Comment:
I assume you guys pass goal in for some purpose right? Maybe just test what instances are detected as goal